project/Dependencies.scala

```diff
@@ -0,0 +1,34 @@
+package akka.projections
+
+import sbt._
+import sbt.Keys._
+
+object Dependencies {
+
+  object Versions {
+    val akka = "2.5.17"
+
+    val scalaTest = "3.0.5"
+    val scalaJava8Compat = "0.9.0"
```

Comment:
use same as Akka, which is 0.8.0 (for bincompat reasons)

BTW, don't specify this dependency at all, it comes with akka-stream

Reply:
I copied it from akka-grpc. We should change there as well. (cc @raboof)

Reply:
created issue https://github.com/akka/akka-grpc/issues/437